### PR TITLE
Feat motor android

### PR DESCRIPTION
**Pull request by mcjcloud**
Fixes to motor needed for querySchema in android.

<sub> Mention [stepsize] in a comment if you'd like to report some technical debt. See examples [here](https://app.stepsize.com/api/demo-pr-redirect).</sub>

### CODE REVIEW

File: scripts/bind.sh [hunk 1]
```diff
@@ -22,7 +22,7 @@ while getopts "iawm" opt; do
       rm -rf ${BUILDDIR}/io.sonr.motor-sources.jar
       cd ${BUILDDIR}
       tar -czvf motor-${VERSION}-android.tar.gz io.sonr.motor.aar
-      rm -rf ${ANDROID_ARTIFACT}
```
Comment (ntindle) [inline scripts/bind.sh:25]: ?

File: pkg/client/query.go [hunk 2]
```diff
@@ -10,41 +10,16 @@ import (
 	"google.golang.org/grpc"
 )
 
-// func (c *Client) QueryAccount(address string) (*at.BaseAccount, error) {
```
Comment (ntindle) [inline pkg/client/query.go:13]: Does speedway use this?

File: pkg/motor/query.go [hunk 3]
```diff
@@ -6,45 +6,10 @@ import (
 
 	mt "github.com/sonr-io/sonr/third_party/types/motor"
 	bt "github.com/sonr-io/sonr/x/bucket/types"
+	st "github.com/sonr-io/sonr/x/schema/types"
 )
 
-func (mtr *motorNodeImpl) QueryBucket(req mt.QueryWhereIsRequest) (*mt.QueryWhereIsResponse, error) {
```
Comment (ntindle) [inline pkg/motor/query.go:11]: Will this break speedway?

File: pkg/client/query.go (hunk 2 above)
Comment (mcjcloud) [reply pkg/client/query.go:13]: It was commented out so I don't think so.

File: scripts/bind.sh (hunk 1 above)
Comment (mcjcloud) [reply scripts/bind.sh:25]: This was deleting the .aar file which I need.

File: pkg/motor/query.go (hunk 3 above)
Comment (mcjcloud) [reply pkg/motor/query.go:11]: Not until speedway upgrades version, it will be a simple rename

**Review by ntindle — APPROVED** [review]

**Review by prdlk — APPROVED** [review]

**Pull request merged**